symphony-bdk-core-invokers/symphony-bdk-core-invoker-api/src/main/java/com/symphony/bdk/core/api/invoker/ApiException.java

```diff
@@ -75,11 +77,15 @@ public boolean isClientError() {
     }
 
     /**
-     * Check if response status is server error or not
+     * Check if response status is server error and not an internal server error
      *
-     * @return true if response status equals or greater than 500, false otherwise
+     * @return true if response status strictly greater than 500, false otherwise
      */
     public boolean isServerError() {
-        return this.code >= HttpURLConnection.HTTP_INTERNAL_ERROR;
+        return this.code > HttpURLConnection.HTTP_INTERNAL_ERROR;
```

Comment:
this means we don't care about the `HTTP_INTERNAL_ERROR`?

Reply:
Yes indeed, in this case we assume it is not worth retrying. This was agreed with @symphony-thibault.